style.css

```diff
@@ -259,6 +250,29 @@ body {
     margin-left: 30px;
   }
 
+  .popup-skills > li {
+    font-family: 'Poppins', 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #3c3a39;
+    border: 1px solid #dfe1e6;
+    margin-left: 10px;
+  }
+
+  .skillset > li {
+    font-family: Inter, sans-serif;
+    margin-right: 12px;
+    padding: 6px;
+    border: 1px solid #8993a4;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
```

Comment:
- [ ] Hi @mailsg, Great effort working on the pop up modal milestone of the portfolio project. but i would like to you to work on something in your pop up mobile view in order to make your work look great and in line with what we have in the Figma template. I notice the pop up card is hitting the wall at the top of the page when popped up i would kindly suggest you give it a margin top to make it align center and look perfect.
You can target the id `#details-popup` and modify this property to this `width: 90%; and margin: 24px 5%;`

- I would also advice you work on making the background becomes blur once the card is popped as seen in the Figma design
- Kindly reduce the space between the two buttons


| Your Design | Figma Design |
| ------------- | ------------- |
| ![san1_LI](https://user-images.githubusercontent.com/78634841/233334706-e4b96942-853b-4ef5-90dc-666fa3310827.jpg) | ![real1](https://user-images.githubusercontent.com/78634841/233334759-6667ec42-2254-4733-a42d-3e5a432282d8.png) |

Reply:
The space is now reduced.